dispatches/models/fossil_case/ultra_supercritical_plant/ultra_supercritical_powerplant.py

```diff
@@ -0,0 +1,1424 @@
+    )
+    m.fs.mix9_to_fwh9 = Arc(
+        source=m.fs.fwh_mixer[8].outlet, destination=m.fs.fwh[8].inlet_1
+    )
+    m.fs.bfp_to_fwh9 = Arc(
+        source=m.fs.bfp.outlet, destination=m.fs.fwh[8].inlet_2
+    )
+
+    # fwh10
+    m.fs.fwh9_to_fwh10 = Arc(
+        source=m.fs.fwh[8].outlet_2, destination=m.fs.fwh[9].inlet_2
+    )
+
+    # boiler
+    m.fs.fwh10_to_boiler = Arc(
+        source=m.fs.fwh[9].outlet_2, destination=m.fs.boiler.inlet
+    )
+
+
+def set_model_input(m):
+
+    # Model inputs / fixed variable or parameter values
+    # assumed in this block, unless otherwise stated explicitly,
+    # are either assumed or estimated for a total power out of 437 MW
+
+    # These inputs will also fix all necessary inputs to the model
+    # i.e. the degrees of freedom = 0
+
+    ###########################################################################
+    #  Turbine input                                                          #
+    ###########################################################################
+    #  Turbine inlet conditions
+    main_steam_pressure = 31125980  # Pa
+    m.fs.boiler.inlet.flow_mol.fix(17854)  # mol/s
+    m.fs.boiler.outlet.pressure.fix(main_steam_pressure)
+
+    # Reheater section pressure drop estimated
+    # for a total power out of 437 MW
+    m.fs.reheater_1.deltaP.fix(-742845)  # Pa
+    m.fs.reheater_2.deltaP.fix(-210952)  # Pa
+
+    # The efficiency and pressure ratios of all turbines were estimated
+    # for a total power out of 437 MW
+    m.fs.turbine[1].ratioP.fix(0.388)
+    m.fs.turbine[1].efficiency_isentropic.fix(0.94)
+
+    m.fs.turbine[2].ratioP.fix(0.774)
+    m.fs.turbine[2].efficiency_isentropic.fix(0.94)
+
+    m.fs.turbine[3].ratioP.fix(0.498)
+    m.fs.turbine[3].efficiency_isentropic.fix(0.94)
+
+    m.fs.turbine[4].ratioP.fix(0.609)
+    m.fs.turbine[4].efficiency_isentropic.fix(0.94)
+
+    m.fs.turbine[5].ratioP.fix(0.523)
+    m.fs.turbine[5].efficiency_isentropic.fix(0.88)
+
+    m.fs.turbine[6].ratioP.fix(0.495)
+    m.fs.turbine[6].efficiency_isentropic.fix(0.88)
+
+    m.fs.turbine[7].ratioP.fix(0.514)
+    m.fs.turbine[7].efficiency_isentropic.fix(0.78)
+
+    m.fs.turbine[8].ratioP.fix(0.389)
+    m.fs.turbine[8].efficiency_isentropic.fix(0.78)
+
+    m.fs.turbine[9].ratioP.fix(0.572)
+    m.fs.turbine[9].efficiency_isentropic.fix(0.78)
+
+    m.fs.turbine[10].ratioP.fix(0.476)
+    m.fs.turbine[10].efficiency_isentropic.fix(0.78)
+
+    m.fs.turbine[11].ratioP.fix(0.204)
+    m.fs.turbine[11].efficiency_isentropic.fix(0.78)
+
+    ###########################################################################
+    #  Condenser section                                         #
+    ###########################################################################
+    m.fs.cond_pump.efficiency_isentropic.fix(0.80)
+    m.fs.cond_pump.deltaP.fix(2313881)
+
+    # Make up stream to condenser
+    m.fs.condenser_mix.makeup.flow_mol.value = -9.0E-12  # mol/s
```

Comment:
Just curious about why the flow rate has been set to a negative number?

Reply:
this has now been updated. the flow is initialized to a small value